### PR TITLE
Add core features documentation as includes

### PR DESCRIPTION
**Pull request by sabbyanandan**
Depends on spring-cloud/spring-cloud-dataflow#684

- ~~includes new structure~~
- includes stream category with relevant reordered sections
- ~~includes task category~~
- includes dashboard category

(_once we finalize structure/content, we can then repeat this across other server implementations for consistency_)

### CODE REVIEW

File: spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/spring-cloud-dataflow-features.adoc [hunk 1]
```diff
@@ -0,0 +1,65 @@
+[[spring-cloud-data-flow-features]]
+== Features
+
+* Orchestrate applications across a variety of distributed modern runtimes including: Cloud Foundry, Apache YARN, Apache Mesos, and Kubernetes
```
Comment (ericbottard) [inline spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/spring-cloud-dataflow-features.adoc]: If all servers (including the "local" one) have the same paragraphs, why not make this 65lines file the thing we include?

File: spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/spring-cloud-dataflow-features.adoc [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+==== Tap a Stream
+include::{dataflow-asciidoc}/streams.adoc#spring-cloud-dataflow-stream-tap-dsl[]
+
+==== Explicit Broker Destinations in a Stream
+include::{dataflow-asciidoc}/streams.adoc#spring-cloud-dataflow-stream-explicit-destination-names[]
+
+==== Directed Graphs in a Stream
+include::{dataflow-asciidoc}/streams.adoc#spring-cloud-dataflow-stream-advanced[]
+
+=== Tasks
```
Comment (cppwfs) [inline spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/spring-cloud-dataflow-features.adoc]: Do we want to include a section on tasks, if they are not supported in CF?

**Comment by cppwfs** [comment]
Its also not rendering correctly.  I'm not seeing anything below:
7.1 Streams
7.1.1 Introduction

I ran the following command : ` ./mvnw clean package -DskipTests -P full -am -pl spring-cloud-dataflow-server-cloudfoundry`

**Comment by ericbottard** [comment]
Actually, I don't understand the `include` directive with `#`s

**Comment by sabbyanandan** [comment]
@cppwfs / @ericbottard:

Submitted: spring-cloud/spring-cloud-dataflow#689

With suggestions in the PR above, I'll directly include `streams.adoc` and `dashboard.adoc` as includes directly. YARN and K8S implementations would include `tasks.adoc` as well.

**Comment by sabbyanandan** [comment]
Changes are submitted; however, this PR requires spring-cloud/spring-cloud-dataflow#869, as a prerequisite.

**Comment by cppwfs** [comment]
Images are not showing up.  i.e.
images/dataflow-dashboard-about.png
images/dataflow-available-apps-list.png

They may have to be copied from the host project. :-(

**Comment by sabbyanandan** [comment]
Thanks for catching that!

I changed all the `/images` link to absolute path (in the core docs), so we can then include images in  server specific docs without having to copy them over and maintain as dups. The fix will be addressed in spring-cloud/spring-cloud-dataflow#683.

**Comment by cppwfs** [comment]
LGTM

**Comment by cppwfs** [comment]
squashed merged

**Pull request closed without merging**